src/UI/Implementation/Component/Button/Button.php

```diff
@@ -99,13 +99,13 @@ public function appendOnClick(Signal $signal) {
 	 * @inheritdoc
 	 */
 	public function withOnHover(Signal $signal) {
-		return $this->addTriggeredSignal($signal, 'hover');
+		return $this->addTriggeredSignal($signal, 'mouseenter');
```

Comment:
The `hover` event is not a native javascript event but available in JQuery to bind event handlers on the `mouseenter` and `mouseleave` events. If you only bind one handler, it is used for both events. In the context of this framework, this would mean that the signal is triggered on mouse enter and leave. We want to trigger a signal just once on the `mouseenter` event. See: https://api.jquery.com/hover/